test/lib/Sslang/Test.hs

```diff
@@ -0,0 +1,85 @@
+{- | Defines some common utilities for writing Hspec tests.
```

Comment:
This test library makes tests much nicer to write and read, and removes the boilerplate due to imports.